docs/blockchain-size-explanation.md

```diff
@@ -1,57 +1,68 @@
 
-Mina replaces the entire blockchain, starting from genesis to any block, with an easily verifiable constant-sized cryptographic proof. Verifying this proof corresponding to a block amounts to verifying [all the transactions up until a few blocks behind the current block](#snarked-ledger). The proof and inputs to the verification function are all in the block that gets gossiped.
-This process of verification applies to all the nodes in Mina network. Let's dig a bit deeper to see how much state is required for different type of nodes in Mina network.
+Mina replaces the entire blockchain, starting from genesis to any block, with an easily verifiable constant-sized cryptographic proof. Verifying this proof corresponding to a block amounts to verifying [all the transactions up until a few blocks behind the current block](#snarked-ledger). The proof and inputs to the verification function are all in the latest block that gets gossiped. Note that due to the definition of "blockchain" above including the term "usable" This definition means in succinct protocol like Mina, we need more than just a proof object coupled with a state full of opaque hashes.
 
-## Light clients
+This process of verification applies to all the nodes in Mina network. Some roles in the network do need extra information to perform their duties, but this is beyond the definition we gave above. Read more about a "Block producer" node in the appendix.
 
-The main functionalities of a light client are 1) to be able to check balances of specific accounts 2) to be able to submit transactions to the network. For this, they need the following information.
+## Data needed for a usable represntation of the blockchain
 
-#### Block
+We wish (1) to be able to check balances of a specific account (2) to be able to submit transactions to the network. For this, a node, we call it a "non-consensus node" needs stored on in memory merely: a protocol state, the account, a merkle path to this account, and a verification key. Importantly, these powerful nodes are not light clients. They have equivalent security to full nodes.
 
-A block in mina respresents the state of the blockchain at a given height. It mainly consists of a blockchain proof, state represented using hashes of various datastructures including the ledger (we call this a protocol state), new transactions, SNARKs for the transactions included in the previous blocks. To verify the current block and all the previous blocks in the chain starting from genesis, one just needs the proof and the protocol state included in the current block (all hail recursive zk-SNARKS!). Then using the verifier function and verification keys, a node can determine whether this state is valid or not.
+This node role is not yet implemented. The protocol will support it, we just need to prioritize the engineering work.
```

Comment:
Should we instead say something along the lines of `it is on the roadmap post mainnet`?

Reply:
Yes, good call